examples/backendpolicy.yaml

```diff
@@ -0,0 +1,15 @@
+kind: BackendPolicy
+apiVersion: networking.x-k8s.io/v1alpha1
+metadata:
+  name: my-app
+spec:
```

Comment:
Shall a backendRefs be added to this example?

Reply:
Added, thanks!